src/agent/common.ts

```diff
@@ -468,6 +455,24 @@ function createMaskFunction(jobEnvironment: agentifm.JobEnvironment): Replacemen
             }
         });
 
+        endpoints.forEach((endpoint: agentifm.ServiceEndpoint) => {
+            if (endpoint.authorization != null && endpoint.authorization['parameters'] != null) {
+                var keys = Object.keys(endpoint.authorization['parameters']);
+                keys.forEach((key: string) => {
+                    var valueToReplace: string = endpoint.authorization['parameters'][key];
```

Comment:
This scenario is similar to an empty secret variable.
Good thing to add would be a null check.